specs/media/v1/mathoid.yaml

```diff
@@ -0,0 +1,128 @@
+# Mathoid - math formula rendering service
+
+swagger: 2.0
+
+paths:
+  /{module:media}/math:
+    post:
+      tags:
+        - Media
+        - Math
+      description: >
+        Renders a TeX formula into its mathematic representation in the given format.
+        Available formats are svg, json and mml.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#unstable).
+      produces:
+        - image/svg+xml
+        - application/mathml+xml
+        - application/json
+      parameters:
+        - name: format
+          in: formData
+          description: The output format; can be svg, json or mml
+          type: string
+          required: false
```

Comment:
We could use swagger's enum stanza here, e.g.

    - name: type
      in: formData
      description: The input type of the given formula; can be tex, inline-tex, mml or ascii
      type: string
      enum:
        - tex
        - inline-tex
        - mml
        - ascii
And it would be rendered in docs with a choice box, which's more convenient.